### ISSUE TITLE
False positive LC0075

### ISSUE THREAD

**Issue by fvet** [StefanMaron/BusinessCentral.LinterCop#874]
@Arthurvdv Would you consider below warning as false positive? The second field of the PK is a decimal field, although I would expect it to accept integer values as well.

Seems a bit odd to change the code from 99999 to 99999**.0**

![image](https://github.com/user-attachments/assets/0dfc17c7-7ef2-4ef0-a2e8-34beb9109875)

**Comment by Arthurvdv** [comment]
You're right, an impliciet conversion from an ```Integer``` to an ```Decimal``` shouldn't be a problem and the rule should allow this.

I believe I've resolved this in the Pre-release [v0.32.1](https://github.com/StefanMaron/BusinessCentral.LinterCop/releases/tag/v0.32.1) version of the LinterCop. Could you verify if it now works as expected?

**Comment by fvet** [comment]
Retested with the pre-release and the issue has been fixed (y)

Thx for the quick fix.
